concent_api/concent_api/settings/base.py

```diff
@@ -210,6 +210,12 @@
 # A global constant defining the length of the time window within which a requestor can verify subtask.
 SUBTASK_VERIFICATION_TIME = 4 * 3600  # 4 hours
 
+# A global constant defining the lenght of the time window within which a requestor must pay
+PAYMENT_DUE_TIME = 12 * 3600  # 12 hours
+
+# A global constant defining the length of the time windows within which a provider can enforce payment
```

Comment:
Not exactly, this is rather a time window within which a provider must wait before enforcing payment.